translations/base-en.yaml

```diff
@@ -316,16 +316,16 @@ ingame:
 shopUpgrades:
     belt:
         name: Belts, Distributor & Tunnels
-        description: Speed +<gain>%
+        description: Speed x<currentMult> → x<newMult>
```

Comment:
I feel like `Speed x1 -> x2` is a bit unclear / doesn't look that nice, maybe we can find a better wording?